tests/conftest.py

```diff
@@ -1130,7 +1131,13 @@ def mock_events(mocked_responses, api_url):
                         "status": "no",
                     },
                 ],
-            }
+            },
+            {
+                "id": "9876543cba",
+                "message_id": None,
```

Comment:
Would this have a None value, or would the `message_id` key be omitted entirely?

Reply:
It would have a None value.